clients/extension/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "__MSG_appName__",
   "version": "5.0.0",
-  "minimum_chrome_version": "72",
+  "minimum_chrome_version": "76",
```

Comment:
idk when this will be, this PR should chill until the Chromium change lands in stable.